src/lurk/cli/tests/mod.rs

```diff
@@ -39,3 +39,22 @@ fn test_lib() {
     let mut repl = Repl::new_native();
     assert!(repl.load_file("lib/tests.lurk".into(), false).is_ok());
 }
+
+#[ignore]
+#[test]
+fn test_demo_files() {
+    set_config(Config::default());
+    let demo_files = [
+        "demo/simple.lurk",
+        "demo/functional-commitment.lurk",
+        "demo/chained-functional-commitment.lurk",
+        "demo/protocol.lurk",
+        "demo/bank.lurk",
+        "demo/mastermind.lurk",
+    ];
```

Comment:
It might be easier to maintain the list of demo files in a single `tests.lurk` as done in `lib` above.

Reply:
Noted. Will keep that in mind